dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -209,7 +209,8 @@
       userAge: current_user.age,
       userType: current_user.user_type,
       isOauth: current_user.oauth?,
-      isPasswordRequired: current_user.encrypted_password.present?
+      isPasswordRequired: current_user.encrypted_password.present?,
+      authenticationOptions: current_user.authentication_options,
```

Comment:
Would this potentially render OAuth tokens on the client? Maybe we should find a way to only include the info we need from each authentication option.